client/state/jetpack-connect/reducer/schema.js

```diff
@@ -1,26 +1,31 @@
 /** @format */
 export const jetpackConnectAuthorizeSchema = {
-	type: 'object',
-	additionalProperties: false,
-	patternProperties: {
-		'^.+$': {
+	anyOf: [
+		{
+			type: 'object',
+			additionalProperties: false,
+			properties: {},
+		},
```

Comment:
If schema validation fails or the timestamp is stale, `{}` is used. Allow matching empty object.